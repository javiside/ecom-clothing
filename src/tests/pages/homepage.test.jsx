import React from "react";
import { shallow } from "enzyme";

import Homepage from "../../pages/homepage";

describe("Homepage", () => {
  it("should mount the component correctly", () => {
    expect(shallow(<Homepage />)).toMatchSnapshot();
  });
});
