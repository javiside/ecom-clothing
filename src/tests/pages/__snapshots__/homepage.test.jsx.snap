// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should mount the component correctly 1`] = `
<div
  className="homepage"
>
  <Connect(Directory) />
</div>
`;
